src/licensedcode/data/licenses/ofl-1.1.LICENSE

```diff
@@ -2,12 +2,13 @@
 key: ofl-1.1
 short_name: OFL 1.1
 name: SIL Open Font License 1.1
-category: Permissive
+category: Copyleft Limited
```

Comment:
@DennisClark are we clear on this?

Reply:
@pombredanne Dennis has already commented on this https://github.com/nexB/scancode-toolkit/pull/3470#discussion_r1283304481 here